.eslintignore

```diff
@@ -0,0 +1,9 @@
+dist/
+docs/
+examples/
+node_modules/
+lib/
+scripts/
+# for disable website but should be done as well
+website/
+*.md
```

Comment:
When you update the comment, can you also add a newline at the end of the file? Makes life easier (internally, we have lints that try to automatically add these).